etc/webapi.xml

```diff
@@ -40,4 +40,11 @@
             <resource ref="anonymous" />
         </resources>
     </route>
+    <!-- Pre-Auth: Create Order hook -->
+    <route url="/V1/bolt/boltpay/order/create" method="POST">
```

Comment:
I don't have a strong feeling but `POST bolt/boltpay/order` implies it creates order so `/create` feels a bit redundant 

https://restfulapi.net/http-methods/#post

Reply:
We already have bolt/boltpay/order/manage endpoint, so this complies with that. It is not a strict standard restful APi, I agree.